packages/contracts-ts/README.md

```diff
@@ -0,0 +1,51 @@
+## Contracts TS
```

Comment:
This package will end up replacing the SDK in many ways.  It's the contract interface to contracts-bedrock and for low maintenance cost it is mostly generated